src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,73 @@
-import React from 'react';
+import React, { useLayoutEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useDispatch } from 'react-redux';
+import { currentTodoSlice } from '../../features/currentTodo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoModal: React.FC<Props> = ({ todo }) => {
+  const [isLoading, setLoading] = useState(true);
+  const [user, setUser] = useState<User | null>(null);
+  const { removeCurrentTodo } = currentTodoSlice.actions;
+
+  const dispatch = useDispatch();
+
+  useLayoutEffect(() => {
+    getUser(todo.userId)
+      .then(setUser)
+      .catch()
```

Comment:
The `catch` block is currently empty. It's a good practice to handle errors properly, perhaps by setting an error state or logging the error to the console.